ui/app/manager/src/components/configuration/or-conf-map/or-conf-map-global.ts

```diff
@@ -0,0 +1,262 @@
+                    </div>
+                </div>
+                <div class="map-style-layers-group">
+                    <span>
+                        <div class="subheader"><or-translate value="configuration.global.mapLayers"></or-translate></div>
+                        <or-translate value="configuration.global.mapImportStyle"></or-translate>
+                        <or-translate class="note" value="configuration.global.mapImportStyleNote"></or-translate>
+                    </span>
+                    <div style="display: flex; gap: 12px; align-items: center">
+                        <or-mwc-input class="input fit-content" type="button" outlined icon="import" 
+                            .label="${i18next.t("configuration.global.import")}"
+                            .disabled="${!this.config.override}" @or-mwc-input-changed="${this.importMapSettings}"
+                        ></or-mwc-input>
+                        <or-conf-json class="input fit-content hide-mobile" .heading="${i18next.t("configuration.global.mapLayers")}"
+                            .config="${hasExternalSource ? this.config.layers.filter(({ id }) => !id.startsWith("or:")) : this.config.layers}"
+                            @saveLocalConfig="${(ev: CustomEvent) => {
+                                if (Array.isArray(ev.detail.value)) {
+                                    if (hasExternalSource) {
+                                        // Set default layers
+                                        this.config.layers =  this.config.layers.filter(({ id }) => id.startsWith("or:"))
+                                        // Push changed layers
+                                        this.config.layers.push(...ev.detail.value);
+                                    } else {
+                                        this.config.layers = ev.detail.value;
+                                    }
+                                    this.notifyConfigChange(this.config);
+                                    this.requestUpdate();
+                                }
+                        }}"
+                        ></or-conf-json>
+                        ${when(hasExternalSource, () => html`
+                            <or-mwc-input class="input fit-content" outlined type="button" icon="undo"
+                                .label="${i18next.t("configuration.global.reset")}"
+                                @or-mwc-input-changed="${this.resetMapSettings}"
+                            ></or-mwc-input>
+                        `)}
+                        </div>
+                </div>
+            </div>
+        `
+    }
+
+    protected notifyConfigChange(config: MapConfig) {
+        this.dispatchEvent(new CustomEvent("change", { detail: config }));
+    }
+
+    protected notifyMapFileChange(e: OrInputChangedEvent) {
+        const file = (e.target as HTMLInputElement)?.files?.[0];
+        if (file.size > this.limit) {
```

Comment:
A few commits ago, this function had a `file is defined` check.
Now, if it is not defined, it will throw an exception.
I'd handle this correctly if "no file has been submitted".